versioning.md

```diff
@@ -0,0 +1,279 @@
+Versioning
+==========
+
+Starting with version 2.0.0, VOLK adapts [Semantic Version](https://semver.org) as its versioning scheme.
+You'll find the full text of Semver 2.0.0 below.
+
+To highlight a few core aspects as relevant to VOLK:
+
+  * Adding a new kernel (e.g. `volk_32fc_prove_riemann_hypothesis`) is a backwards-compatible functionality addition. That requires increasing the MINOR version (clause 7).
+  * Adding a new proto-kernel (e.g. `volk_32fc_prove_riemann_hypothesis_a_neon`) is _also_ a backwards-compatible functionality addition, and hence also increases MINOR version. It's appreciated to bundle these.
+  * Fixing a bug in a proto-kernel that doesn't change the function signature nor intended behavior only requires MINOR to be increased.
```

Comment:
Should this be `PATCH` and not `MINOR`? Seems to match the definition below of the former, not the latter.

Reply:
Indeed!